src/components/App/App.jsx

```diff
@@ -67,6 +66,8 @@ class App extends React.Component {
     const github = new GitHub()
     github.getCurrentUser().then(user => {
       this.setState({ user })
+      DefaultFilters.init(user.login)
+      this.setState({ filters: Filters.findAll() })
```

Comment:
Since this function is getting longer, maybe add a new method in App called `onUserLoad` and move lines 68-70 into it?